modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -0,0 +1,143 @@
+
+To use the help, type the following command:
+`./service mesh help`
+
+image::service-mesh-cli-adapter-help.png[CLI Help for Adapter Commands]
+
+== Provision Service Mesh Adapter
+
+Before	you can start managing your API using Service Mesh, you must prepare your namespace by provisioning a Service Mesh Adapter. Provisioning prepares the namespace for the Service Mesh configuration.
+
+To provision the Anypoint Service Mesh adapter:
+ 
+. Determine the adapter size based on the plan.
+For more information, see...
+. From the Service Mesh installer download location in the Command window, type the following command to create the adapter:
++
+[source,text,linenums]
+----
+./service-mesh adapter create \--name=anypoint-service-mesh-adapter \--namespace=marketing \
+
+--size=medium \--clientId=<replace_with_clientId> \
+
+--clientSecret=<replace_with_clientSecret> \
+
+--platformUri=https://anypoint.mulesoft.com
+----
++	
+Where:
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. For the Beta exercise, use “anypoint-service-mesh-adapter” for this parameter. | Required | N/A
+| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. For the Beta exercise, use “marketing” for this parameter | Optional | default
+| size | The size of the adapter based on the number of CPUs and memory to use | Optional | Medium
+| clientId | The client ID of the application | Required | N/A
+| clientSecret | The client secret of the application | Required | N/A
+| platformUri | The URL of the host where Anypoint Platform is installed | Optional | https://anypoint.mulesoft.com
+|===
++
+. Verify the adapter status:
+`./service-mesh adapter list`
++
+image::provision-service-mesh-adapter-output.png[Example Output]
++
+. Review adapter logs to see detailed information about Kubernetes events and adapter transactions.
+`./service-mesh adapter logs --name=<adapter_name> --namespace=<namespace>`
+
+
+== Bind your service to an API on API Manager
+After you set up your namespace, you must bind the service with an existing API to the adapter that you created.
+
+To bind the service with an API, perform the following steps:
+
+. If not exists, create your API in API Manager.
+For more information, see https://docs.mulesoft.com/api-manager/2.x/create-instance-task
+. If your applications were already running in the namespace, restart your application:
+`$ kubectl delete pod -n marketing -l app=campaigns`
++
+. Verify that the application Kubernetes service port is named following the requirements requested by Istio: https://istio.io/docs/setup/additional-setup/requirements/ .
+. From the Service Mesh installer download location in the Command window, type the following command to bind the adapter with an application:
++
+[source,text,linenums]
+----
+./service-mesh binding create \
```

Comment:
@priyaayyar As discussed, This section is missing critical pieces 

- how to manage an API with Anypoint Service Mesh in API Manager
- How to get an API Id